libr/flag/flag.c

```diff
@@ -562,6 +562,7 @@ R_API RFlagItem *r_flag_set(RFlag *f, const char *name, ut64 off, ut32 size) {
 				r_skiplist_insert (f->by_off, flagsAtOffset);
 			}
 		}
+		R_FREE (flagsAtOffset);
```

Comment:
if you remove this in here it will fuck the skiplist, that's why there are tons of XX

Reply:
Changes reverted